src/info.c

```diff
@@ -388,10 +388,10 @@ static int option_parse_applet(void *pctx, struct apk_db_options *dbopts, int op
 	case 'r':
 		ctx->subaction_mask |= APK_INFO_RDEPENDS;
 		break;
-	case 'I':
```

Comment:
Also, it looks like the arguments --install-if and --rinstall-if were reversed before